datahub-web-react/src/app/ingest/source/ExecutionRequestDetailsModal.tsx

```diff
@@ -1,48 +1,147 @@
-import { message, Modal, Typography } from 'antd';
-import React from 'react';
+import { DownloadOutlined } from '@ant-design/icons';
+import { Button, message, Modal, Typography } from 'antd';
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import { useGetIngestionExecutionRequestQuery } from '../../../graphql/ingestion.generated';
+import { ANTD_GRAY } from '../../entity/shared/constants';
+import { downloadFile } from '../../search/utils/csvUtils';
 import { Message } from '../../shared/Message';
+import IngestedAssets from './IngestedAssets';
+import {
+    getExecutionRequestStatusDisplayColor,
+    getExecutionRequestStatusDisplayText,
+    getExecutionRequestStatusIcon,
+} from './utils';
 
 const Section = styled.div`
     display: flex;
     flex-direction: column;
     padding-bottom: 12px;
 `;
 
-const SectionHeader = styled(Typography.Text)`
+const SectionHeader = styled(Typography.Title)`
     &&&& {
         padding: 0px;
         margin: 0px;
         margin-bottom: 12px;
     }
 `;
 
+const HeaderSection = styled.div``;
+
+const StatusSection = styled.div`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const IngestedAssetsSection = styled.div`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const LogsSection = styled.div`
+    padding-top: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
 type Props = {
     urn: string;
     visible: boolean;
     onClose: () => void;
 };
 
 export const ExecutionDetailsModal = ({ urn, visible, onClose }: Props) => {
+    const [showExpandedLogs, setShowExpandedLogs] = useState(false);
     const { data, loading, error } = useGetIngestionExecutionRequestQuery({ variables: { urn } });
     const output = data?.executionRequest?.result?.report || 'No output found.';
 
+    useEffect(() => {
+        if (output.length > 100) {
+            setShowExpandedLogs(false);
+        }
+    }, [output, setShowExpandedLogs]);
+
+    const downloadLogs = () => {
+        downloadFile(output, `exec-${urn}.log`);
+    };
+
+    const logs = (showExpandedLogs && output) || output.slice(0, 100);
+    const result = data?.executionRequest?.result?.status;
+
+    const ResultIcon = result && getExecutionRequestStatusIcon(result);
+    const resultColor = result && getExecutionRequestStatusDisplayColor(result);
+    const resultText = result && (
+        <Typography.Text style={{ color: resultColor, fontSize: 14 }}>
+            {ResultIcon && <ResultIcon style={{ marginRight: 4 }} />}
+            {getExecutionRequestStatusDisplayText(result)}
+        </Typography.Text>
+    );
+    const resultSummaryText =
+        (result && (
+            <Typography.Text type="secondary">
+                Ingestion {result === 'SUCCESS' ? 'successfully completed' : 'completed with errors'}.
```

Comment:
nit: could use constant SUCCESS defined in src/app/ingest/source/utils.ts!

Reply:
Boom money!